### DISCUSSION TITLE
archive list and delete list

### DISCUSSION THREAD

**Discussion by shiresn** [mgmeyers/obsidian-kanban#218] [Ideas & Feature Requests]
I'm just want to get familiar with all options before start using this plugin, In the Kanban view, after click "more option" on one list, I can see options "Archive list" and "delete list", if I click "archive list", the list disappears from the kanban view just like delete list. but can it be unarchived or be seen in the markdown view(just like archived cards)? if not what's the difference between "delete list" and "archive list"?

**Comment by mgmeyers** [comment]
Hey @shiresn!

Currently, archived cards go into a hidden list that is only viewable in markdown mode. The benefit of this is that archived cards will sill show up in searches as well as the backlinks and tag panes. In contrast, deleted cards actually get removed from the markdown file.

In the future, there will be a way to view and search archived cards in kanban mode.

**Reply by shiresn** [reply]
Hello @mgmeyers , 

is archived lists go into some hidden place just like archived cards?

**Reply by mgmeyers** [reply]
Yeah. The list isn't saved, but all the cards in the list are

**Reply by zzfmyway** [reply]
Hi @mgmeyers , 

Is it possible to keep the archived list and maintain the structure between archived list and its archived cards?

Currently, archived cards would be moved under level 2 title ##Archived.

Alternatively, is it possible to save archived list like this:

```
## Archived
### Archived List 
- [x] archived card 1
- [x] archived card 2 
```

In this way, the archived cards would remain in its original list, and the list can be used to backlink to somewhere else for reference.

**Reply by Medullitus** [reply]
> Hey @shiresn!
> 
> Currently, archived cards go into a hidden list that is only viewable in markdown mode. The benefit of this is that archived cards will sill show up in searches as well as the backlinks and tag panes. In contrast, deleted cards actually get removed from the markdown file.
> 
> In the future, there will be a way to view and search archived cards in kanban mode.

Hello @mgmeyers  

Thank you for the plugin. But even after 3 years, yes 3 years! it is still concerning that you haven't implemented this application. Am I wrong?
